### PR TITLE
Corrected some minor typos

### PR DESCRIPTION
**Pull request by 1nathanliang**
Thank you for this reference, I was just following this guide for fMRI data and wanted to fix a few minor typos I noticed along the way.

### CODE REVIEW

**Review by Remi-Gau — APPROVED** [review]
Fantastic. Thanks for catching those. :rocket:

**Comment by Remi-Gau** [comment]
@all-contributors please add @1nathanliang  for bug

File: src/folders_and_files/files.md [hunk 1]
```diff
@@ -3,12 +3,12 @@
 BIDS has a standardized way of naming files that tries to implement the
 following principles:
 
--   Do not white spaces in file names
+-   Do not includwe white spaces in file names
```
Comment (dorahermes) [inline src/folders_and_files/files.md:6]: There is a typo in this sentence (includwe should not have a w)

Comment (Remi-Gau) [reply src/folders_and_files/files.md:6]: :see_no_evil: 

will fix it

Comment (Remi-Gau) [reply src/folders_and_files/files.md:6]: fixed by #279 
thanks @dorahermes

Comment (1nathanliang) [reply src/folders_and_files/files.md:6]: Oops 🙈 my bad!

**Review by dorahermes — COMMENTED** [review]
lgtm

**Pull request merged**